your_loggers/your_workflow_dacx.py

```diff
@@ -0,0 +1,37 @@
+from temporalio import workflow
+
+"""dacx
+You can log from a Workflow using Python's standard library, by importing the logging module `logging`.
+
+Set your logging configuration to a level you want to expose logs to.
+
+```python
+logging.basicConfig(level=logging.INFO)
+```
+
+Then in your Workflow, set your [`logger`](https://python.temporal.io/temporalio.workflow.html#logger) and level on the Workflow. The following example logs the Workflow.
+dacx"""
+
+
+@workflow.defn
+class GreetingWorkflow:
+    def __init__(self) -> None:
+        self._greeting = "<no greeting>"
+
+    @workflow.run
+    async def run(self, name: str) -> None:
+        workflow.logger.info(f"Workflow information: {name}")
```

Comment:
This is not workflow information like `workflow.info()` is, this is just the parameter